fix: Add explanation for copying files (#86)
ray-project/ray-llm: docs/kuberay/deploy-on-eks.md

```sh

> # Step 7.1: Log in to the head Pod
> export HEAD_POD=$(kubectl get pods --selector=ray.io/node-type=head -o custom-columns=POD:metadata.name --no-headers)
> kubectl cp ../../serve_configs $HEAD_POD:/home/ray/serve_configs # If it does not exist
> kubectl exec -it $HEAD_POD -- bash

Reviewer: ```suggestion
# Step 7.1: Copy Serve configs and log in to the head Pod
export HEAD_POD=$(kubectl get pods --selector=ray.io/node-type=head -o custom-columns=POD:metadata.name --no-headers)
kubectl cp ../../serve_configs $HEAD_POD:/home/ray/serve_configs # Copy the configs if they don't exist in the pod
kubectl exec -it $HEAD_POD -- bash
```

Author: Thank you for reviewing and correcting it!
